include/swift/Basic/LangOptions.h

```diff
@@ -443,6 +443,12 @@ namespace swift {
     // FrontendOptions.
     bool AllowModuleWithCompilerErrors = false;
 
+    // Allow @package attribute in the file.
+    bool AllowPackageDeclaration = false;
```

Comment:
I think a little more information about when either of these flags should be set would be helpful. Also this should be plural (i.e. `Declaration*s*`).